mesmerize_viz/_cnmf.py

```diff
@@ -788,10 +895,19 @@ def _set_data(self, data_arrays: Dict[str, np.ndarray]):
 
         self._eval_controller.set_limits(self._cnmf_obj)
 
-    def _euclidean(self, source, target, event, new_data):
+    def click_event(self, ev):
```

Comment:
rename to "image_clicked"

just have this find the nearest graphic to the click event, determine the index of that graphic and then call `set_component_index()`